npx projen build is broken in latest

npx projen build
.........
 PASS  lib/__tests__/new.test.js (14.711 s)

Test Suites: 32 passed, 32 total
Tests:       1161 passed, 1161 total
Snapshots:   44 passed, 44 total
Time:        30.471 s
Ran all test suites.
🤖 build » test » eslint | eslint --ext .ts,.tsx --fix --no-error-on-unmatched-pattern src src/__tests__ build-tools
🤖 build » package | jsii-pacmak
[jsii-pacmak] [WARN] Exception occurred, not cleaning up 
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /Users/pgollucci/src/github.com/projen/projen/node_modules/yargs/package.json
    at new NodeError (node:internal/errors:278:15)
    at throwExportsNotFound (node:internal/modules/esm/resolve:321:9)
    at packageExportsResolve (node:internal/modules/esm/resolve:546:3)
    at resolveExports (node:internal/modules/cjs/loader:473:36)
    at Function.Module._findPath (node:internal/modules/cjs/loader:513:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:911:27)
    at Function.resolve (node:internal/modules/cjs/helpers:98:19)
    at Object.resolveDependencyDirectory (/Users/pgollucci/src/github.com/projen/projen/node_modules/jsii-pacmak/lib/util.js:18:33)
    at /Users/pgollucci/src/github.com/projen/projen/node_modules/jsii-pacmak/lib/dependency-graph.js:44:36
    at Array.map (<anonymous>)
🤖 Task "build » package" failed when executing "jsii-pacmak" (cwd: /Users/pgollucci/src/github.com/projen/projen)
npm ERR! code 1
npm ERR! path /Users/pgollucci/src/github.com/projen/projen
npm ERR! command failed
npm ERR! command sh -c projen build

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pgollucci/.npm/_logs/2021-01-09T16_15_10_801Z-debug.log

git log
* '6b55631 - (HEAD -> master, tag: v0.14.4, origin/master, origin/HEAD) chore(release): 0.14.4 (17 hours ago) <Auto-bump>'

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^10.17.0",
     "@types/semver": "^7.3.4",
-    "@types/yargs": "^15.0.10",
+    "@types/yargs": "^15.0.12",
     "@typescript-eslint/eslint-plugin": "^4.3.0",
     "@typescript-eslint/parser": "^4.3.0",
     "eslint": "^7.14.0",
@@ -70,7 +70,7 @@
     "semver": "^7.3.2",
     "xmlbuilder2": "^2.4.0",
     "yaml": "^1.10.0",
-    "yargs": "^16.1.1"
+    "yargs": "^16.2.0"
   },
   "bundledDependencies": [
     "@iarna/toml",

